addon/components/lt-head.hbs

```diff
@@ -48,11 +48,11 @@
                 extra=this.extra
                 sortIcons=this.sortIcons
                 resizeOnDrag=this.resizeOnDrag
-                click=(action "onColumnClick" column)
-                doubleClick=(action "onColumnDoubleClick" column)
-                onColumnResized=(action "onColumnResized")
-                onColumnDrag=(action "onColumnDrag")
-                onColumnDrop=(action "onColumnDrop")
+                click=(fn this.columnClick column)
+                doubleClick=(fn this.columnDoubleClick column)
+                columnResized=this.columnResized
```

Comment:
Is there a reason we wanted to rename these instead of keeping `on` in the name?

Reply:
I had to change the name several places because the internal method had the same name as the passed parameter. They were both `this.method` when converted to a native class generating infinite loop errors. 

 On later updates I started prefixing these rather than dropping the `on`.  I missed this one when cleaning up.  I will fix this

Reply:
Fixed names

Reply:
@maxwondercorn right, but the duplicate named method should be deleted. They were just wrappers to call the actions, so they are not needed.